changelog.txt

```diff
@@ -1,4 +1,11 @@
 ---------------------------------------------------------------------------------------------------
+Version: 4.1.2
+Date: ???
+  Changes:
+    - Start with Bob's grey belts when they are enabled
+    - Update starting items for Sea Block
+    - Prevent Sea Block rock chest from spawning
+---------------------------------------------------------------------------------------------------
```

Comment:
Please remove this. I will update the Changlog file myself on update and credit you. :-)

Reply:
Done